content/en/docs/concepts/security/index.md

```diff
@@ -12,565 +12,543 @@ aliases:
+following diagram shows the architecture.
 
-{{< image width="80%" link="./architecture.svg" caption="Istio Security Architecture" >}}
+{{< image width="75%"
+    link="./arch-sec.svg"
+    caption="Security Architecture"
+    >}}
 
 In the following sections, we introduce the Istio security features in detail.
 
 ## Istio identity
 
-Identity is a fundamental concept of any security infrastructure. At the beginning of a service-to-service communication,
-the two parties must exchange credentials with their identity information for mutual authentication purposes.
-On the client side, the server's identity is checked against the [secure naming](/docs/concepts/security/#secure-naming)
-information to see if it is an authorized runner of the service.
-On the server side, the server can determine what information the client can access based on the
-[authorization policies](/docs/concepts/security/#authorization-policy),
-audit who accessed what at what time, charge clients based on the services they used,
-and reject any clients who failed to pay their bill from accessing the services.
-
-In the Istio identity model, Istio uses the first-class service identity to determine the identity of a service.
-This gives great flexibility and granularity to represent a human user, an individual service, or a group of services.
-On platforms that do not have such identity available,
-Istio can use other identities that can group service instances, such as service names.
-
-Istio service identities on different platforms:
-
-- **Kubernetes**: Kubernetes service account
-
-- **GKE/GCE**: may use GCP service account
-
-- **GCP**: GCP service account
-
-- **AWS**: AWS IAM user/role account
-
-- **On-premises (non-Kubernetes)**: user account, custom service account, service name, Istio service account, or GCP service account.
-  The custom service account refers to the existing service account just like the identities that the customer's Identity Directory manages.
-
-### Istio security vs SPIFFE
-
-The [SPIFFE](https://spiffe.io/) standard provides a specification for a framework capable of bootstrapping and issuing identities to services
-across heterogeneous environments.
-
-Istio and SPIFFE share the same identity document: [SVID](https://github.com/spiffe/spiffe/blob/master/standards/SPIFFE-ID.md) (SPIFFE Verifiable Identity Document).
-For example, in Kubernetes, the X.509 certificate has the URI field in the format of
-`spiffe://<domain>/ns/<namespace>/sa/<serviceaccount>`.
-This enables Istio services to establish and accept connections with other SPIFFE-compliant systems.
-
-Istio security and [SPIRE](https://spiffe.io/spire/), which is the implementation of SPIFFE, differ in the PKI implementation details.
-Istio provides a more comprehensive security solution, including authentication, authorization, and auditing.
+Identity is a fundamental concept of any security infrastructure. At the
```

Comment:
This section is inconsistent with the similar material in the glossary. In the glossary it talks about Istio assigning identifies to workloads but here it says they are given to service identities.
I suggest fixing this material to be consistent with the glossary version, and then shortening the glossary version to just have a short summary and a link to this section in the concepts.
https://preliminary.istio.io/docs/reference/glossary/#identity

Reply:
@frankbu I am changing the content in the section to use `workload` instead of `service`. That said, fully aligning all content to use the workload terminology is outside the scope of this PR. That also applies to reworking the identity glossary entry.